packages/__globalPreview/left-sidebar.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <script src="https://unpkg.com/react@18/umd/react.production.min.js"></script>
+    <script src="https://unpkg.com/react-dom@18/umd/react-dom.production.min.js"></script>
+    [component://static-header]
+  </head>
+  <body class="dark:su-bg-black-true dark:su-text-white">
+    <a id="skiplinks" href="#main-content" class="skiplink">Skip to content</a>
+    <header
+      class="report-header--story report-header su-pb-[11rem] md:su-pb-[17.7rem] lg:su-pb-[19.3rem] su-bg-black"
+    >
+      <div
+        class="su-bg-digital-red su-fixed"
+        style="height: 100px; width: 100%"
+      >
+        <button
+          id="toggle"
+          class="su-p-7 su-border-2 su-text-white su-border-white dark:su-border-black-true dark:su-text-black-true"
+        >
+          Dark/Light Mode Toggle
+        </button>
```

Comment:
This is so handy by the way. I'm going to steal your idea and add it to the Campaign preview template 😆

Reply:
I think I'll put it on all of the preview files eventually.